package/examples/discord_bot.js

```diff
@@ -1,38 +1,36 @@
-//Discord.js v13.8.0
-import {Client, Intents} from 'discord.js';
-import HenrikDevValorantAPI from 'unofficial-valorant-api';
-const VAPI = new HenrikDevValorantAPI();
+// Discord.js v13.8.0
+import { Client, Intents } from "discord.js"
+import _VAPI from "unofficial-valorant-api"
+const VAPI = new _VAPI()
 
 const client = new Client({
-    intents: [Intents.FLAGS.GUILDS],
-});
+    intents: [ Intents.FLAGS.GUILDS ],
+})
 
-client.on('ready', () => console.info(`Bot ${client.user.tag} online.`));
+client.on("ready", () => console.info(`Bot ${client.user.tag} online.`))
 
-client.on('interactionCreate', async interaction => {
+client.on("interactionCreate", async interaction => {
     if (interaction.isCommand()) {
         switch (interaction.commandName) {
-            case 'mmr': {
-                await interaction.deferReply({ephemeral: true});
-                const mmr_data = await VAPI.getMMR({
-                    version: 'v1',
-                    region: 'eu',
-                    name: interaction.options.getString('name'),
-                    tag: interaction.options.getString('tag'),
-                });
-                if (mmr_data.error)
+            case "mmr": {
+                await interaction.deferReply({ ephemeral: true })
+                const mmr_data = await VAPI.getMMR(interaction.options.getString("name"), interaction.options.getString("tag"), "na")
```

Comment:
Why changing region from `eu` to `na`?

If it was just for you you should change it back.

Reply:
Ok